FusionIIIT/applications/research_procedures/views.py

```diff
@@ -675,12 +675,7 @@ def add_financial_outlay(request,pid):
 def inbox(request):
  
     user_designation= request.session.get('currentDesignationSelected')
-    if user_designation == 'Assistant':
-        user_designation = 'Assistant Professor'
-    
-    if user_designation == 'CSE':
-        user_designation= 'CSE HOD'
-    
```

Comment:
if these lines are removed how is `user_designation` actually set for the field?